components/mintmaker/base/mintmaker-config.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: mintmaker-config
+  namespace: mintmaker
+data:
+  config.js: |
```

Comment:
The file must be named `renovate.json` and be in JSON format. The controller will patch the following keys

- platform
- endpoint
- username
- gitAuthor

Here is an example configmap we can use at this moment: https://gist.github.com/qixiang/915389b2c26b8ca5dbe479e2a5038bb0

Reply:
Ok, I've taken something from there, I don't think we want to keep them all based on what I've understood by stakeholders. But I've taken what I think it makes sense.